Makefile

```diff
@@ -13,10 +13,10 @@ dialyzer.umbrella:
 	mix dialyzer
 
 dialyzer.projects:
-	$(foreach dir, $(dialyzer_dirs), cd projects/$(dir) && mix dialyzer && cd ../..;)
+	$(foreach dir, $(dialyzer_dirs), cd projects/$(dir) && mix dialyzer; cd ../..;)
```

Comment:
It was a minor issue I ran into and figured I'd just fix. The problem was that the `cd ../..` wouldn't run if one of the mix commands failed/errored, which would then lead to a Make error along the lines of "no such directory projects/lexical_shared" because the next loop would still be in the first project directory.

I'm not sure this fix is correct though; it might hide errors if the first mix command fails but the last one succeeds. I'll just revert.